src/assets/js/connection.js

```diff
@@ -102,6 +102,23 @@ export default class Connection {
           med.h.showNotification(notification);
         }
       }
+      else if (data.subEvent == "video-mute") {
+        if (data.muted) {
+          var notification = data.username + " is hiding";
+          med.h.showNotification(notification);
+          const widget = document.getElementById(`${data.socketId}-widget`);
+          if (widget) {
+            widget.hidden = true;
+          }
+        } else {
+          var notification = data.username + " is back"
+          med.h.showNotification(notification);
```

Comment:
We are now also showing people video being muted as they are disappear from the screen. When audio is not muted they are still able to talk